tests/network-tests/src/fixtures/council/ElectCouncilFixture.ts

```diff
@@ -160,10 +172,20 @@ export class ElectCouncilFixture extends BaseQueryNodeFixture {
     await this.api.untilCouncilStage('Idle')
 
     const councilMembers = await api.query.council.councilMembers()
+    this.councilMembersIds = councilMembers.map((m) => m.membershipId)
     assert.sameMembers(
-      councilMembers.map((m) => m.membershipId.toString()),
+      this.councilMembersIds.map((id) => id.toString()),
       candidatesToWinIds
     )
+
+    // change accounts to known accounts
+    const oldCouncilMemberAccounts = await this.getCouncilMembersControllerAccounts()
```

Comment:
Instead of having to fetch those accounts here, you can query for the member's `rootAccount` in `updateMemberControllerAccount`, I think that will be simpler. (`update_accounts` extrinsic should be called using member's `rootAccount`, not `controllerAccount`, although in the integration tests they are usually the same)